ember-cli/2017-08/august-17.md

```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2017-08-17
+
+## Attendees
+
+## Agenda
```

Comment:
Discuss using [@std/esm](https://github.com/standard-things/esm) in ember-cli (and possibly by default for addons). 

Good writeup in the introductory blog post: https://medium.com/web-on-the-edge/es-modules-in-node-today-32cff914e4b.